react/components/MaybeAuth.tsx

```diff
@@ -28,12 +30,12 @@ export default class MaybeAuth extends PureComponent<Props, State> {
   }
 
   public isAuthenticatedPage() {
-    return this.props.pages[this.props.page].login
+    return this.props.pages[this.props.page].login || this.props.pages[this.props.page].auth
   }
 
   public redirectToLogin() {
     const pathName = window.location.pathname.replace(/\/$/, '')
-    if (this.props.page !== 'store/login' && pathName !== LOGIN_PATH) {
+    if (!loginPages.includes(this.props.page) && pathName !== LOGIN_PATH) {
```

Comment:
I think changing `page` to `.` will probably break some apps' assumptions 😢 This reaaally feels like a major

Reply:
Yes. Sorry.